lib/bamboo/attachment.ex

```diff
@@ -2,24 +2,36 @@ defmodule Bamboo.Attachment do
   @moduledoc """
   """
 
-  defstruct filename: nil, content_type: nil, path: nil, data: nil
+  defstruct filename: nil, content_type: nil, path: nil, data: nil, content_id: nil
 
   @type t :: %__MODULE__{
           path: nil | String.t(),
           filename: nil | String.t(),
           content_type: nil | String.t(),
-          data: nil | binary()
+          data: nil | binary(),
+          content_id: nil | String.t()
         }
 
   @doc ~S"""
   Creates a new Attachment
 
+  context_id can be use to embed an image, attach it and reference in the message body by
```

Comment:
`context_id` -> `content_id`
`can be use` -> `can be used`
`and reference` -> `and reference it`